src/main.js

```diff
@@ -372,6 +372,26 @@ class ImageKitUppyPlugin extends Plugin {
             }
             formData.append("publicKey", this.opts.publicKey);
             formData.append("file", file.data);
+            if (this.opts.createdBy) {
+              if (
+                !this.opts.createdBy.userId ||
+                !this.opts.createdBy.userId.trim().length > 0
+              ) {
+                this.uppy.log(
+                  `UserId passed is invalid ${this.opts.createdBy.toString()}`
```

Comment:
`Missing or invalid userId` would be better. 

Also `${this.opts.createdBy.toString()}` would just print `"[object Object]"` in most cases. Let's remove it.